test/e2e/page-objects/flows/install-test-snap.flow.ts

```diff
@@ -11,27 +11,37 @@ import { WINDOW_TITLES } from '../../helpers';
  *
  * @param driver - WebDriver instance used to interact with the browser.
  * @param buttonName - The name of the button to click.
- * @param withWarning - Whether the installation will have a warning dialog, default is false.
+ * @param options - Optional parameters.
+ * @param options.withWarning - Whether the installation will have a warning dialog, default is false.
+ * @param options.withExtraScreen - Whether there is an extra screen after the Ok, defaults to false.
  */
 export async function openTestSnapClickButtonAndInstall(
   driver: Driver,
   buttonName: keyof typeof buttonLocator,
-  withWarning = false,
+  options: {
+    withWarning?: boolean;
+    withExtraScreen?: boolean;
+  } = {},
 ) {
+  const { withWarning = false, withExtraScreen = false } = options;
   const snapInstall = new SnapInstall(driver);
   const snapInstallWarning = new SnapInstallWarning(driver);
   const testSnaps = new TestSnaps(driver);
   await testSnaps.openPage();
   await testSnaps.scrollAndClickButton(buttonName);
   await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
   await snapInstall.check_pageIsLoaded();
-  await snapInstall.clickNextButton();
+  await snapInstall.clickConnectButton();
   await snapInstall.clickConfirmButton();
   if (withWarning) {
     await snapInstallWarning.check_pageIsLoaded();
     await snapInstallWarning.clickCheckboxPermission();
     await snapInstallWarning.clickConfirmButton();
   }
-  await snapInstall.clickNextButton();
```

Comment:
again incorrect naming caused by the use of the same selector for different actions